coalib/results/AbsolutePosition.py

```diff
@@ -1,7 +1,8 @@
 from coalib.results.TextPosition import TextPosition
-from coala_utils.decorators import enforce_signature
+from coala_utils.decorators import enforce_signature, generate_ordering
 
 
+@generate_ordering('position')
```

Comment:
that means there's equality between the same positions in different texts? That doesn't make too much sense to me :/